library/std/build.rs

```diff
@@ -16,6 +16,9 @@ fn main() {
     } else if target.contains("freebsd") {
         println!("cargo:rustc-link-lib=execinfo");
         println!("cargo:rustc-link-lib=pthread");
+        if env::var("RUST_STD_FREEBSD_12_ABI").is_ok() {
+            println!("cargo:rustc-cfg=freebsd12");
+        } else { unreachable!()}
```

Comment:
that's not quite true, but also not terribly important. Just panic with a helpful message?

Reply:
oops, I was testing, this definitely should not be there